ksqldb-streams/src/test/java/io/confluent/ksql/execution/streams/StreamSelectKeyBuilderTest.java

```diff
@@ -75,7 +79,7 @@
 
   private static final LogicalSchema RESULT_SCHEMA = LogicalSchema.builder()
       .withRowTime()
-      .keyColumn(SchemaUtil.ROWKEY_NAME, SqlTypes.BIGINT)
+      .keyColumn(ColumnName.of("BOI"), SqlTypes.BIGINT)
```

Comment:
I thought we should never get to the situation where a key and value column have the same name? (or is this now temporarily possible during the partition by step? that seems not ideal)

Reply:
The key and rowkey columns are copied into the value... so yes, we _always_ have duplicates in the schema used during processing...